datalake/src/main/java/com/sequenceiq/datalake/job/SdxClusterStatusCheckerJob.java

```diff
@@ -66,35 +77,44 @@ private void handleAmbiguousSdx(StackStatusV4Response stack, SdxCluster sdx) {
 
     private void handleStoppedSdx(StackStatusV4Response stack, SdxCluster sdx) {
         if (stack.getStatus().isAvailable() && stack.getClusterStatus().isAvailable()) {
-            sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.RUNNING, ResourceEvent.SDX_STOP_FINISHED, "", sdx);
+            sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.RUNNING, ResourceEvent.SDX_START_FINISHED, "", sdx);
             logStateChange(DatalakeStatusEnum.STOPPED, DatalakeStatusEnum.RUNNING);
         }
-        if (stack.getStatus().equals(Status.DELETE_COMPLETED)) {
-            sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.STACK_DELETED, ResourceEvent.SDX_CLUSTER_DELETION_FINISHED, "", sdx);
-            jobService.unschedule(String.valueOf(sdx.getId()));
-            logStateChange(DatalakeStatusEnum.RUNNING, DatalakeStatusEnum.STACK_DELETED);
+        setDeleteCompleted(stack, sdx);
+        setDeleteFailed(stack, sdx);
+    }
+
+    private void handleFailedSdx(StackStatusV4Response stack, SdxCluster sdx) {
+        if (stack.getStatus().isAvailable() && stack.getClusterStatus().isAvailable()) {
+            sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.RUNNING, ResourceEvent.SDX_REPAIR_FINISHED, "", sdx);
```

Comment:
`SDX_REPAIR_FINISHED` message can be a little bit misleading as if it's not coming from repair failed